lib/vulcan.js

```diff
@@ -400,8 +410,14 @@ Vulcan.prototype = {
                 dom5.append(ownerDomModule, domTemplate);
               }
               dom5.remove(tag);
-              // put the style at the top of the dom-module's template
-              this.prepend(domTemplate.childNodes[0], style);
+              if (!lastPolymerExternalStyle) {
+                // put the style at the top of the dom-module's template
+                this.prepend(domTemplate.childNodes[0], style);
+              } else {
+                // put this style behind the last polymer external style
+                dom5.insertBefore(domTemplate.childNodes[0], nextSibling(lastPolymerExternalStyle), style);
```

Comment:
Seems like it would be easier to have the promises resolve with the style, and then in the `then` take the array of styles from `Promise.all` and append them in one go?

Or else just reverse the array of links to make inserting easier.